lms/templates/problem_notifications.html

```diff
@@ -3,8 +3,8 @@
 <%! from django.utils.translation import ugettext as _ %>
 
 <div class="notification ${notification_type} ${'notification-'}${notification_name}"
-     tabindex="-1">
-    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span>
+     tabindex="-1" ${'style="display: none"'if notification_name == 'error' else '' } >
```

Comment:
@cahrens I tested this locally and it seems to work just as well.  So let me know if you want me to push the change.